### ISSUE TITLE
Freezing PS3 when Refreshing XML if certain Scan options are disabled!

### ISSUE THREAD

**Issue by Dracrius** [aldostools/webMAN-MOD#248]
I'm having issues disabling certain scan options as it seems to cause the PS3 to hang on a refresh. At first with all default scan options I had no issues but once getting things setup how I like I disabled PS2, Bluray and DVD as I dont use any of them. I also disabled all usb's but usb03 which I use for my isos. After this a refresh of the xml/ game list seemed to crash EVERY time and seemed to fail to make it to ROM's but picked up almost everything else. 

I went through and reenabled every usb with no change. Then I reenabled ps2 options as well but it would still crash. I finally reenabled bluray and dvd at once as I was getting a little frustrated but this did allow for a successful scan with everything enabled. I got a little fedup and stopped there so idk if you have to have them all enabled or if its one of the last two (bluray and DVD) that finally allowed a successful scan without crashing.

**Comment by aldostools** [comment]
Thank you for your feedback. I will try to reproduce this error and check what causes the issue.

**Comment by aldostools** [comment]
Hello @Dracrius 
I couldn't reproduce the error removing the DVD and BD options and re-scanning the XML. 
The groups were removed properly and the other groups showed the content as they should.

If you have any clue of how I could reproduce the error, please let me know.
For the moment, the solution is to keep these options selected even if you don't use them.

**Comment by Dracrius** [comment]
After further testing on my side it only crashes when I disable scanning for Blu-ray. Any combination of ps2 and DVD disabled caused no problems while trying to refresh the XML. But no matter the combination if I disabled Blu-ray it would hang at what I think is the very end or literally the step before searching for ROMs as on reboot it will show the desired sub menus but ROMs is always empty after the freeze. I didn't have a chance to test changing it back tonight but the only other thing I've really changed was the RAM setting to 4 you mentioned while I was having trouble scanning for psx titles. My ROMs folder is also quite full with 300+ ROMs and a scan takes atleast a full minute almost a minute and a half to complete.

**Comment by Dracrius** [comment]
Is there any test you'd suggest to narrow down whats causing it on my PS3?

**Comment by aldostools** [comment]
ATM I don't have any suggestion. Try asking in PSX-PLACE forum, maybe other users have ideas to suggest.

If it is working with BD & DVD options selected, just keep them enabled.

**Comment by szymonszyszy** [comment]
> Is there any test you'd suggest to narrow down whats causing it on my PS3?

Hello, I am Szymon. and I also have that same problem but different but maybe the solution is to reinstall the Full Version of webMAN Mod if you have the Full Version and have been maybe using version 4.89, well I think you should reinstall because that will maybe gonna fix this problem.

**Comment by aldostools** [comment]
Yes, that's one of the first suggestions in the troubleshooting guide:
https://github.com/aldostools/webMAN-MOD/wiki#troubleshooting

**Comment by Joonie86** [comment]
No further feedback
